### DISCUSSION TITLE
Como cadastrar minha empresa pra implementar o pix i.e. ter nossos próprios client credentials

### DISCUSSION THREAD

**Discussion by LucasBadico** [bacen/pix-api#402] [Q&A]
Olá, estamos construindo uma solução para pequenos vendedores, nossa intenção é nascer com o pix como default, mas não encontro em lugar nenhum onde posso fazer o onboarding.

**Comment by ninrod** [comment]
@LucasBadico , boa noite. 

você precisa de um PSP recebedor que implemente a API Pix. Normalmente são empresas que detém a "conta" do usuário recebedor (a.k.a, os "pequenos vendedores").

**Comment by rubenskuhl** [comment]
> Olá, estamos construindo uma solução para pequenos vendedores, nossa intenção é nascer com o pix como default, mas não encontro em lugar nenhum onde posso fazer o onboarding.

Se juntar o acima com o que você colocou no subject, você precisa se tornar um PSP. A estrutura necessária para isso é muito grande, mas se você acha mesmo que tem como, o formulário de adesão fica em:
https://www.bcb.gov.br/content/estabilidadefinanceira/pix/Formulario_adesao_nao_autorizacao_funcionamento_IN104.docx

Agora, quando você cair na real e ver que faz mais sentido trabalhar junto com um PSP já estabelecido, eu tento manter uma lista atualizada em #76 .

**Comment by ninrod** [comment]
Obrigado @rubenskuhl
